nvflare/edge/device/android/app/src/main/java/com/nvidia/nvflare/app/controllers/FlareRunnerController.kt

```diff
@@ -74,6 +75,17 @@ class FlareRunnerController(
             "methods" to listOf("cnn", "xor")
         )
     
+    /**
+     * Create a connection to the NVFlare server
+     */
+    private fun createConnection(): Connection {
```

Comment:
we need to support HTTPS as well, but that could be in a different PR, now I see android sdk we are hardcoded HTTP

Reply:
yes, that is in a separate one